corehq/apps/domain/views.py

```diff
@@ -1269,9 +1269,12 @@ def parent_pages(self):
 
     @property
     def steps(self):
+        edition_name = u" (%s)" % self.edition_name if self.edition_name else ""
         return [
             {
-                'title': _("1. Select a Plan%s") % (" (%s)" % self.edition_name if self.edition_name else ""),
+                'title': _(u"1. Select a Plan%(edition_name)s" % ({
+                    "edition_name": edition_name
+                })),
```

Comment:
Sorry, my previous comment wasn't very clear. This still has the same problem. You want something like this:
```py
_(u"1. Select a Plan{edition_name}").format(edition_name=edition_name)
```